packages/wallet-connect-sdk-svelte/src/WcSdkStore.ts

```diff
@@ -0,0 +1,248 @@
+import { writable, type Writable, type Readable, derived } from 'svelte/store'
+import SignClient from '@walletconnect/sign-client'
+import WcSdk from '@cityofzion/wallet-connect-sdk-core'
+import type {
+  NetworkType,
+  Method,
+  Neo3Invoker,
+  Neo3Signer,
+  BuiltTransaction,
+  CalculateFee,
+  ContractInvocationMulti,
+  InvokeResult,
+  RpcResponseStackItem,
+  DecryptFromArrayResult,
+  EncryptedPayload,
+  SignMessagePayload,
+  SignedMessage,
+  NetworkVersion,
+  WalletInfo,
+} from '@cityofzion/wallet-connect-sdk-core'
+import type { SessionTypes, SignClientTypes } from '@walletconnect/types'
+
+export interface IWalletConnectStore extends Neo3Invoker, Neo3Signer {
+  /**
+   * The WalletConnect Library
+   */
+  signClient: Readable<SignClient | null>
+
+  /**
+   * The current WalletConnect connected session
+   */
+  session: Readable<SessionTypes.Struct | null>
+
+  /**
+   * returns if the session is connected
+   */
+  readonly isConnected: Readable<boolean>
```

Comment:
It will be too difficult to maintain the documentation if one of the libraries has a different usage than the others. Today we have a single "usage" documentation, and both "core" and "react" libraries respect it.
So, instead of `isConnected` being a getter it need to be a method. The devs will still be able to use `session` variable to check if they are connected instead of using the method.
Reference: https://github.com/CityOfZion/wallet-connect-sdk/blob/main/USAGE_GUIDE.md#check-if-the-user-is-connected